lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -3395,6 +3395,132 @@ class SPIRVComplexFloatInst
 _SPIRV_OP(ComplexFMulINTEL)
 _SPIRV_OP(ComplexFDivINTEL)
 #undef _SPIRV_OP
+
+class SPIRVGatherScatterINTELInstBase : public SPIRVInstTemplateBase {
```

Comment:
Maybe SPIRVMaskedGatherScatterINTELInstBase is a more accurate name? (Just a nit)

Reply:
Applied